war/pom.xml

```diff
@@ -415,7 +411,7 @@ THE SOFTWARE.
                 <artifactItem>
                   <groupId>org.jenkins-ci.modules</groupId>
                   <artifactId>sshd</artifactId>
-                  <version>3.0.3</version>
+                  <version>3.1.0</version>
```

Comment:
Needed to avoid this error:

```
java.io.IOException: Failed to load: Jenkins GIT server Plugin (git-server 1.11)
 - Update required: SSH server (sshd 3.0.3) to be updated to 3.1.0 or higher
        at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:994)
        at hudson.PluginManager.dynamicLoad(PluginManager.java:926)
Caused: java.io.IOException: Failed to install git-server plugin
        at hudson.PluginManager.dynamicLoad(PluginManager.java:940)
        at hudson.model.UpdateCenter$InstallationJob._run(UpdateCenter.java:2217)
Caused: java.io.IOException: Failed to dynamically deploy this plugin
        at hudson.model.UpdateCenter$InstallationJob._run(UpdateCenter.java:2221)
        at hudson.model.UpdateCenter$DownloadJob.run(UpdateCenter.java:1867)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at hudson.remoting.AtmostOneThreadExecutor$Worker.run(AtmostOneThreadExecutor.java:121)
        at java.base/java.lang.Thread.run(Thread.java:829)
```